src/APIFunctions/2DPrinting.js

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
-import { PrintApiResponse } from './ApiResponses';
-import { RPC_API_URL } from '../config/config.json';
+import {
+  PrintApiResponse,
+  ApiResponse } from './ApiResponses';
```

Comment:
have ApiResponse on it's own line so it looks like
```js
import {
  PrintApiResponse,
  ApiResponse
} from './ApiResponses';
```

Reply:
```suggestion
import {
  PrintApiResponse,
  ApiResponse
} from './ApiResponses';
```